tests/store/api/status/test_analyses_to_delivery_report.py

```diff
@@ -87,3 +63,26 @@ def test_outdated_analysis(analysis_store, helpers):
 
     # THEN this analyses should not be returned
     assert len(analyses) == 0
+
+
+def test_analyses_to_upload_delivery_reports(analysis_store, helpers):
+    """Tests extraction of analyses ready for delivery report upload"""
+
+    # GIVEN an analysis that has a delivery report generated
+    pipeline = Pipeline.BALSAMIC
+    timestamp = datetime.now()
```

Comment:
I think there is test fixture already for timestamp_now that you can import reuse